code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc

```diff
@@ -0,0 +1,373 @@
+== Monitor and Fix Issues in Your Scans
+
+toc::[]
+
+Prisma Cloud performs periodic scans on each integrated repository of the Version Control System (VCS) and CI/CD to find infrastructure misconfigurations.
```

Comment:
```suggestion
Prisma Cloud performs periodic scans on each integrated repository of onboarded Version Control Systems (VCS) and CI/CD pipelines to find infrastructure misconfigurations, open source vulnerabilities, license compliance violations, and exposed secrets.
```

We may need to highlight that VCS scans are periodic, CI/CD are on a trigger event